### DISCUSSION TITLE
Suggest adding the word `debug` to README

### DISCUSSION THREAD

**Discussion by nickdos** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#143] [Ideas]
In the previous version the `debug` option allowed the drawn coordinates to be displayed. I assumed it would be in v.2 but the README makes no mention of it at all. It wasn't until I laboriously read the entire page that I found the "Hints" section that explained how to do this. I suggest you add a line or extra text in that section: "(replaces the `debug` config option from v1.x)" or similar. That way a quick search for `debug` will find it for your existing users.

I also note that the new "press and hold" the play button has a problem on Chrome. It shows a browser alert but the **text is not selectable** and therefore there is no way to copy the text! So I have to read and retype the coordinates it shows.

**Comment by PiotrMachowski** [comment]
Thanks for the idea, I have added `debug` in documentation :)

Selection is also printed in browser's console, which is much easier to copy from (added info in documentation as well)

**Reply by nickdos** [reply]
Great, thanks for the quick response.
